lessons/git2/lesson.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: page
+title: 'More on Git'
+visible: true
+---
+<!-- change visible to true if you want it on the site -->
+
+# An Intermediate Look at Git + GitHub #
+
+ - **Authors**: Elliott Sales de Andrade
+ - **Research field**: Seismology
+ - **Lesson topic**: Version control (Git)
+ - **Lesson content URL**: <https://github.com/UofTCoders/studyGroup/tree/gh-pages/lessons/git2>
+
+## Recap: ##
+
+* Configure your git client (`git config user.name` + `user.email`)
+* Create a git repository (`git init`)
+* Start tracking a file with git (`git add`)
+* Commit changes to the history (`git commit`)
+* Check what's going on (`git status`)
+* Compare a file with the one in the history (`git diff`)
+* Look into your history (`git log`)
+
+Please follow along in the [slides](slides.pdf) for the remaining information.
```

Comment:
You reference the pdf without the pdf being included in the PR. Mind including it?